projects/packages/search/src/dashboard/components/record-meter/index.jsx

```diff
@@ -16,10 +16,6 @@ import './style.scss';
 /**
  * Generate Record Meter showing how many records the user has indexed
  *
```

Comment:
Did we lose this jsdoc in the rebase perhaps? Shall we add it back?

Reply:
Oops, we did! 🙈